package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "publish-if-not-published",
-  "version": "1.0.2",
+  "version": "1.1.0",
```

Comment:
Will the added behaviour break things for anyone relying on that now? Is it a breaking change or are we fixing a bug?

Reply:
Yeah, potentially we are. Will go up to 2.0.0 👍